src/sentry/api/endpoints/user_notification_details.py

```diff
@@ -100,22 +83,12 @@ def put(self, request: Request, user) -> Response:
                     status=status.HTTP_400_BAD_REQUEST,
                 )
 
-            type = get_type_from_user_option_settings_key(key)
-            if type:
-                NotificationSetting.objects.update_settings(
-                    ExternalProviders.EMAIL,
-                    type,
-                    get_option_value_from_int(type, int(value)),
-                    user_id=user.id,
-                )
-            else:
-                # Legacy user options which does not include weekly report
-                user_option, _ = UserOption.objects.get_or_create(
-                    key=USER_OPTION_SETTINGS[key]["key"],
-                    user=user,
-                    project_id=None,
-                    organization_id=None,
-                )
-                user_option.update(value=str(int(value)))
+            user_option, _ = UserOption.objects.get_or_create(
+                key=USER_OPTION_SETTINGS[key]["key"],
+                user=user,
+                project_id=None,
+                organization_id=None,
+            )
+            user_option.update(value=str(int(value)))
```

Comment:
Nit: Is explicit conversions to `str` necessary? 

Feel free to ignore, I see it's just copy of the old code

Reply:
@vartec not sure but I don't want to touch it lol.